swagger/v1/api_spec.json

```diff
@@ -4240,117 +4181,6 @@
           }
         }
       },
-      "NPQParticipantCompletedDeclarationAttributes": {
```

Comment:
I think there are a lot of changes in this spec, can we limit to only extended? Unless you needed to cleanup something?